### DISCUSSION TITLE
User Sign-up

### DISCUSSION THREAD

**Discussion by rchibnik** [oqtane/oqtane.framework#4591] [Q&A] [answered]
Is there any module that allows end users to sign up?

User provides name/password, email and maybe two factor authorization to create a new user account and authenticate it with an email and text verification?

Thanks for any help or suggestions.

**Comment by ChadSolberg** [comment, marked as answer]
This is all built into the framework. In site settings you can allow for user registration (this is the signup feature. Note you will need to go to Admin / System Info and restart the application to see this as an option.

OAuth is also an option. Go to admin / User Management  -> Settings  -- scroll down to External Login Settings and enter config there.

**Reply by rchibnik** [reply]
Thank you both @sbwalker  and @ChadSolberg .

I dont know how I missed this and glad its included.

**Comment by sbwalker** [comment]
As @ChadSolberg mentioned above... login as a host user, use Control Panel and navigate to Admin Dashboard, select User Management, select Settings tab:

![image](https://github.com/user-attachments/assets/9e8c7e21-d92c-4155-9276-8b8e781e4e93)

There are many options related to user registration, etc... which you can configure per site.
